recipe/meta.yaml

```diff
@@ -1,23 +1,24 @@
 {% set name = "mosaic" %}
-{% set version = "0.1.0" %}
+{% set version = "0.1.1" %}
 
 package:
   name: {{ name|lower }}
   version: {{ version }}
 
 source:
   url: https://github.com/E3SM-Project/mosaic/archive/{{ version }}.tar.gz
-  sha256: 6393c3bc58257c1214903e30f19cdf7688ba6262215088655e77d9c1e852d57e
+  sha256: 2908dad7df50789be7108a34520a6a4e5b49aad140877e755ece513d0b21c746
 
 build:
-  script: {{ PYTHON }} -m pip install . --no-deps -vv
+  script: {{ PYTHON }} -m pip install . --no-deps -vv --no-build-isolation
   noarch: python
   number: 0
 
 requirements:
   host:
     - python >=3.9
     - pip
+    - setuptools
```

Comment:
@andrewdnolan, we should add this explicitly to the `pyproject.toml` as well.  Here's an example from `mache`:
https://github.com/E3SM-Project/mache/blob/main/pyproject.toml#L69-L71

Reply:
I actually [already had this](https://github.com/E3SM-Project/mosaic/blob/54de949f5ea7415116552cabce922ca5ae7aaa52/pyproject.toml#L29-L31) in the `pyproject.toml`, I think it's in the example from setuptools I followed. I didn't quite follow what it was used for, so I didn't delete it. 

From looking at the `mache` example, looks like `setuptools` doesn't need to be in the requirements a specified in the `pyproject.toml` but should be in the `dev-environment.txt` file. I'll add that the a `#Building` section

Reply:
I see, Sorry. I didn't check.  Then I think we're set -- we just forgot to add it explicitly here in the conda package.

Yes, it should also be in `dev-environment.txt`.